src/main/java/kostka/moviecatalog/controller/MovieCatalogController.java

```diff
@@ -0,0 +1,30 @@
+package kostka.moviecatalog.controller;
+
+import kostka.moviecatalog.entity.Movie;
+import kostka.moviecatalog.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class MovieCatalogController {
+    private MovieService movieService;
+
+    @Autowired
+    public MovieCatalogController(final MovieService movieService) {
+        this.movieService = movieService;
+    }
+
+    @GetMapping("movies/all")
```

Comment:
Doporuceni: Lepsi je, kdyz ten prefix, ktery je stejny pro cely kontroler das do reqest mappingu  controlleru, ale je to jen drobnost ;)

Reply:
Dá se to vůbec dát do mapping kontrolleru v anotaci @Restcontroller?